src/platforms/zulip/triagebot.md

```diff
@@ -94,12 +94,20 @@ You can also define aliases to add additional labels to refer to same target tea
 
 ```toml
 [ping.cleanup-crew]
-alias = ["llvm", "llvms"]
+alias = ["cleanup", "cleanups", "shrink", "reduce", "bisect"]
 message = """\
 message content...
 """
 ```
 
+This will allow the command `@rustbot ping cleanup-crew` to be understood with all the aliased variants, ex.:
+
+```text
+@rustbot ping cleanups
```

Comment:
I'd suggest to have `cleanup` since it's the most used alias from what I've seen
```suggestion
@rustbot ping cleanup
```